### DISCUSSION TITLE
Enabled klevu search | How to disable quick-search.js

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#81] [3. Integrations]
Hi,

We are using klevu search in our website. So I am getting below error on pages from the quick-search.js

![image](https://user-images.githubusercontent.com/9527439/228441080-e037d292-0fce-4586-8437-a2ebb1ea2d3b.png)

I think Can we disable the quick-search.js from the default one.

You can also check it here : https://eeupgrade.mothercare.co.id/catalogsearch/result/?q=baby

Thanks
Rutul

**Comment by roma84** [comment]
Hi, 
here you can see how Mirasvit disables it: https://github.com/breezefront/module-breeze-mirasvit-search-ultimate/blob/master/view/frontend/web/js/autocomplete.js#L8-L10

**Reply by rutulritz** [reply]
Hi,

Thanks for the suggestion. But here I am using **kleu** search and the autocomplete suggestions also rendering from klevu template. 

The mini search form suggestions is working fine for all pages on page load. However it is not working on **Turbo** visited pages[Ex. from megamenu links].
You can check here.  https://eeupgrade.mothercare.co.id

Can you please guide me to fix this?

Thanks

**Reply by vovayatsyuk** [reply]
You need to split the klevu script into two parts to make it compatible with Turbo:

1. The part that loads dependencies should be declared in the head.
2. The part that initializes the script on the search field should be declared in footer.

See: https://breezefront.com/docs/custom-javascript#miscellaneous-html

![image](https://user-images.githubusercontent.com/306080/230625540-4745b476-f103-4ea9-a858-7f678da8909c.png)

**Comment by sandipklevu** [comment]
I have made changes to the below layout and js on breeze default file that works so far. 
Created based on Stubs file.
Ref. https://github.com/breezefront/module-breeze/blob/master/Console/Command/Stubs.php#L69-L92

`File: Swissup/BreezeQuickSearchDisable/view/frontend/layout/breeze_default.xml`

```
<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
      xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <body>
        <referenceBlock name="breeze.js">
            <arguments>
                <argument name="bundles" xsi:type="array">
                    <item name="default" xsi:type="array">
                        <item name="items" xsi:type="array">
                            <item name="Swissup_BreezeQuickSearchDisable/js/disable" xsi:type="array">
                                <item name="path" xsi:type="string">Swissup_BreezeQuickSearchDisable/js/disable</item>
                            </item>
                        </item>
                    </item>
                </argument>
            </arguments>
        </referenceBlock>
    </body>
</page>
```

`File: Swissup/BreezeQuickSearchDisable/view/frontend/web/js/disable.js`

```
define([
    'jquery',
], function ($) {
    'use strict';

    //Disabling the default quickSearch behavior
    $.mixin('quickSearch', {
        component: false
    });
});
```
